cibyl/config.py

```diff
@@ -0,0 +1,64 @@
+"""
+#    Copyright 2022 Red Hat
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+"""
+import logging
+import os
+from collections import UserDict
+from typing import Iterable
```

Comment:
Why do we need to import Iterable just for the docstring? Is it so when we run sphinx it will be able to tell the type?

Reply:
Luckily, Sphinx takes care of this problem. It allows you to reference a class without having to import, so by moving to it this is no longer a problem.